code/datums/supplypacks.dm

```diff
@@ -528,7 +528,7 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	access = access_armory
 	group = "Security"
 
-/datum/supply_packs/security/armory/knight //why seperate them
+/datum/supply_packs/security/armory/knight

```

Comment:
Wait this typepath, does
/datum/supply_packs/security
/datum/supply_packs/security/armory
both actually exist?